docs/guide/mqtt.md

```diff
@@ -2,6 +2,8 @@
 
 You have access to almost all of the [Z-Wave JS APIs](https://zwave-js.github.io/node-zwave-js/#/README) (and more) via MQTT.
 
+The placeholder `<mqtt_name>` in the mqtt topic is the name configured in the MQTT settings.
```

Comment:
I'm not sure if here is a good place or under Setup -> MQTT where the other wildcards are described

Reply:
Maybe it makes more sense to have it in setup mqtt